code-challenges.test.js

```diff
@@ -48,4 +139,44 @@ const dataArray1 = ["array", "object", "number", "string", "Boolean"]
 const dataArray2 = ["string", "null", "Boolean", "string", "undefined"]
 // Expected output: ["array", "object", "number", "string", "Boolean", "null", "undefined"]
 
+//------------------------------------ My Test Code ------------------------------------
+
+// pseudo code:
+// Using jest test template to test a function "uniqueCombinedArrays"
+// it "takes two arrays as arguments. It combines them and filters out any duplicate values"
+// expect(uniqueCombinedArrays(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+// input: since there isn't a function to test, there is no input and I expect a fail (red)
+// output: should output a ReferenceError: uniqueCombinedArrays is not defined
+
+describe ("uniqueCombinedArrays", () => {
+  it("takes two arrays as arguments. It combines them and filters out any duplicate values", () => {
+    expect(uniqueCombinedArrays(dataArray1, dataArray2)).toEqual(["array", "object", "number", "string", "Boolean", "null", "undefined"])
+    })
+  })
+
+// output:
+// FAIL  ./code-challenges.test.js
+// ReferenceError: uniqueCombinedArrays is not defined
+
 // b) Create the function that makes the test pass.
+
+// pseudo code:
+// what's needed? A function "uniqueCombinedArrays" that takes two arrays as arguments. It combines them and filters out any duplicate values.
+// use the spread operator to merge the arrays
+// create a variable to hold the new array use .filter to remove duplicates
+// input: two arrays
+// output: combined array with no duplicate values
+// expected result:  PASS
+
+const uniqueCombinedArrays = (array1, array2) => {
+  let comboArray = [...array1, ...array2]
+  comboArray = comboArray.filter((value, index) => {
+    return comboArray.indexOf(value) === index
+  })
+  return comboArray
+}
+
```

Comment:
Great use of the spread operator and `.filter()` to get rid of duplicates. As a challenge, how could you use the spread operator to allow for as many arguments as you wanted to be passed into your function? Great work!